### ISSUE TITLE
How to train for 20 epochs

### ISSUE THREAD

**Issue by Pritam-N** [switchablenorms/DeepFashion_Try_On#83]
The results are pretty good and the adaptive preservation/ generation module that you used is helping a lot. I am trying to train the model in atr dataset for my academic research. Can you please help me in understanding the training process. You have mentioned training each network separately. I am not able to completely understand how that will work. Also, can you let me know if it will be possible to train all the networks together.

**Comment by LZQhardworker** [comment]
Hi,
You only need to download the dataset and the training code, and run python train.py
Then everything works fine. The networks are trained respectively. (Gradient are not flowed end2end)
